src/vaadin-combo-box-dropdown-wrapper.html

```diff
@@ -328,7 +328,7 @@
        * @return {String}
        */
       getItemLabel(item) {
-        let label = item ? this.get(this._itemLabelPath, item) : undefined;
+        let label = item && this._itemLabelPath ? this.get(this._itemLabelPath, item) : undefined;
```

Comment:
This is better than before but the intended logic still doesn't work correctly.
1. Open "Filtering with a Remote Data Source" (It uses actual objects as items)
1.1. Open the combo box once so the items are listed
2. Set `itemLabelPath` to undefined (Now this produces an error message but the labels are not updated (to the intended toString() output))
3. Set `itemLabelPath` to "link", which is a valid item subproperty -> The error goes away but the items still show old values